packages/react-server-test-pages/gulpfile.babel.js

```diff
@@ -0,0 +1,22 @@
+const eslint = require("gulp-eslint");
+const gulp = require("gulp");
+const babel = require("gulp-babel");
+const logging = require("react-server-gulp-module-tagger");
+
+gulp.task("travis-ci", ["build", "eslint"]);
+
+gulp.task("build", () => gulp.src("src/**/*.js")
+	.pipe(logging())
+	.pipe(babel())
+	.pipe(gulp.dest("lib/"))
+);
```

Comment:
Okay, got it.  I had this thing totally inside out.  Patch inbound that uses `react-server-cli` as intended.